lantern/git/homework.py

```diff
@@ -97,15 +107,20 @@ def is_word_in_text(word: str, text: str) -> bool:
         >>> False
 
     """
-    pass
+    return word in text
 
 
 def some_loop_exercise() -> list:
     """
     Use loop to create list that contain int values from 0 to 12 except 6 and 7
     """
-    pass
 
+    a = []
+    for i in range(13):
+        a.append(i)
+    a.remove(6)
+    a.remove(7)
+    return a
```

Comment:
to many programe code, you can do this shorter:
exepshen = [6, 7]
return[i for i in range(13) if i is not in exepshen]